src/components/ChosenOne/ChosenOne.css

```diff
@@ -37,7 +37,7 @@ iframe {
 
 .details {
   background-color: black;
-  opacity: 70%;
+  opacity: .75;
```

Comment:
OMG !! This is a big thing to know... just changing that it will work in the deploy link?